### PR TITLE
Detector UX, tab and copy style and cleanup

### PR DESCRIPTION
**Pull request by frrrances**

### CODE REVIEW

File: admin/src/ConflictDetectionReporter.js [hunk 1]
```diff
@@ -26,36 +26,80 @@ const STATUS = {
 const STYLES = {
   container: {
     position: 'fixed',
-    fontFamily: 'sans-serif',
+    fontFamily: 'fa5-proxima-nova,"Helvetica Neue",Helvetica,Arial,sans-serif',
```
Comment (mlwilkerson) [inline admin/src/ConflictDetectionReporter.js]: @frrrances we're not loading this proxima-nova font anywhere in this plugin (unless you added it?) So I think the only viewers who will see it in this typeface will be those who happen to have it installed in their local desktop font manager, which is probably just you, Dave and Brian ;-) Is that what you expect?

File: admin/src/ConflictDetectionScannerSection.js [hunk 2]
```diff
@@ -27,37 +27,40 @@ export default function ConflictDetectionScannerSection() {
...
-      of your window (only you can see it) while it runs for 10 minutes. You can
-      then browse your site -- especially pages that are having trouble -- to
-      catch the conflicts in the scanner. Then return to this page to review and
-      manage any Font Awesome versions found.
-    </p>
-    <div>
+      <p>Enable the scanner below and a box will appear in the bottom corner
+      of your window (that only you can see) while it runs for 10 minutes. Then browse
+      your site, especially to the pages having trouble, to
+      catch any Slimers - *ahem* - conflicts in the scanner. Then come back here to review and
```
Comment (mlwilkerson) [inline admin/src/ConflictDetectionScannerSection.js]: In our call earlier, I noticed that the "ahem" was line-wrapping in the middle of the joke. I think the joke might be more enjoyable if it didn't wrap. So maybe something equivalent to:
`<span style="{{ whiteSpace: "nowrap" }}>Slimers - *ahem* - conflicts</span>`?

Comment (frrrances) [reply admin/src/ConflictDetectionScannerSection.js]: Yup! 💯

File: admin/src/ConflictDetectionReporter.js (hunk 1 above)
Comment (frrrances) [reply admin/src/ConflictDetectionReporter.js]: Good catch - I'm going to leave the rest of the stack, but remove that super special one... thanks.

**Pull request merged**